Updated abilities data

IdcNoob opened the pull request.

IdcNoob committed:
Updated abilities data

Moones commented:
If you change Meld attribute "DamageString" to "BonusDamageString" the damage calculation for it will break, so either leave it on "DamageString" or change it also in AbilityDamage class.
I dont think Kunkka Torrent is a nuke, i mean nuke = damage burst which can be used for killstealing, ye torrent can be used for lasthitting low hp enemy but its not a nuke.
Thanks for the big update tho ;)

IdcNoob committed:
Updated abilities data

Changed meld damage to "DamageString".
Removed some nukes which have additional delay, but then spells like assassinate or sun strike shouldnt be nukes too :D
